server/.babelrc

```diff
@@ -1,24 +1,19 @@
 {
   "presets": [
     "@babel/preset-react",
-    "@babel/preset-typescript",
     [
       "@babel/preset-env",
       {
-        "corejs": {
-          "version": "3",
-          "proposals": true
-        },
         "targets": {
-          "node": "12"
-        },
-        "useBuiltIns": "usage"
+          "node": "14"
+        }
       }
-    ]
+    ],
+    "@babel/preset-typescript"
   ],
   "plugins": [
-    "transform-class-properties",
-    "tsconfig-paths-module-resolver",
+    "babel-plugin-transform-typescript-metadata",
```

Comment:
To support outputting of decorator metadata to Reflect, this is key to how sequelize-typescript is able to generate database models from type definitions